frontend/openchat-client/src/openchat.ts

```diff
@@ -4512,152 +4513,165 @@ export class OpenChat extends OpenChatAgentWorker {
         }
     }
 
-    private async loadChats() {
-        try {
-            const init = this._liveState.chatsInitialised;
-            chatsLoading.set(!init);
-
-            const updateRegistryTask = !init ? this.updateRegistry() : undefined;
-
-            const chatsResponse = await this.sendRequest({
-                kind: "getUpdates",
-            });
-
-            if (!init || chatsResponse.anyUpdates) {
-                if (updateRegistryTask !== undefined) {
-                    // We need the registry to be loaded before we attempt to render chats / events
-                    await updateRegistryTask;
-                }
+    private async loadChatsResponse(
```

Comment:
Maybe `processChatsResponse`?
Because at this point the chats response has already been loaded.

Reply:
I've gone with `handleChatsResponse` to mirror the `handleEventsResponse` that we already have.